src/leap/common/events/server.py

```diff
@@ -14,31 +14,27 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
-
-
 """
 The server for the events mechanism.
 """
-
-
 import logging
 import platform
-import txzmq
 
 from leap.common.zmq_utils import zmq_has_curve
 
 from leap.common.events.zmq_components import TxZmqServerComponent
 
+import txzmq
+
```

Comment:
no reason, actually the previous order was better. might be that I removed and had to put it in place again.